dataland-frontend/src/components/pages/UploadEuTaxonomyDataForNonFinancials.vue

```diff
@@ -16,16 +16,17 @@ import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper.vu
 import CreateEuTaxonomyForNonFinancials from "@/components/forms/CreateEuTaxonomyForNonFinancials.vue";
 import CompanyInformation from "@/components/pages/CompanyInformation.vue";
 import TheFooter from "@/components/general/TheFooter.vue";
-import UploaderRoleWrapper from "@/components/wrapper/UploaderRoleWrapper.vue";
+import AuthorizationWrapper from "@/components/wrapper/AuthorizationWrapper.vue";
 import BackButton from "@/components/general/BackButton.vue";
```

Comment:
We seem to have lost again a lot of * associated with mandatory fields

Please check for all three reports upload pages if not doing anything and pressing submit data takes you to properly starred mandatory fields.

![image](https://github.com/d-fine/Dataland/assets/35459207/37d1acae-a186-4bb0-abca-d5bb8d0e7244)

Reply:
has been addressed